### PR TITLE
use webpack

### PR DESCRIPTION
**Pull request by Floozutter**
Bundling with webpack is working!

To be honest, I had no idea what I was doing as I wrote `webpack.config.js`. I'll probably have to refactor that later.

### CODE REVIEW

File: baseline/dev-tools/local-server/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            title: "Flanker Task",
+            filename: "flanker/index.html",
+            chunks: ["flanker"],
+        }),
+        new HtmlWebpackPlugin({
+            filename: "index.html",
+            template: path.join(__dirname, "public/index.html"),
+            chunks: [],
+        }),
+    ],
```
Comment (Floozutter) [inline baseline/dev-tools/local-server/webpack.config.js:24]: These `HtmlWebpackPlugin` objects generate HTML files.

https://webpack.js.org/plugins/html-webpack-plugin/

File: baseline/dev-tools/local-server/webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+            {
+                test: /\.css$/i,
+                use: ["style-loader", "css-loader"],
+            },
+            {
+                test: /\.(png|svg|jpg|jpeg|gif)$/i,
+                type: 'asset/resource',
+            },
+        ],
+    },
```
Comment (Floozutter) [inline baseline/dev-tools/local-server/webpack.config.js:40]: These rules determine how to handle imports.

https://webpack.js.org/configuration/module/

File: baseline/dev-tools/local-server/webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    resolve: {
+        modules: [path.join(__dirname, "node_modules")],
+    },
```
Comment (Floozutter) [inline baseline/dev-tools/local-server/webpack.config.js:43]: This determines how imports find modules.

https://webpack.js.org/configuration/resolve/

File: baseline/client/flanker/index.html [hunk 4]
```diff
@@ -2,13 +2,6 @@
 <html>
```
Comment (noah10) [inline baseline/client/flanker/index.html:2]: Is this file being used for anything any more? If not, let's delete it.

**Review by noah10 — CHANGES_REQUESTED** [review]
As you mentioned, we'll definitely want to think more about how we bundle these (if we're lucky they'll be small enough to bundle all in one), but this is fine for now. Other than (potentially) the need to delete baseline/client/flanker/index.html, this looks good - thanks for getting the webpack stuff set up and working!

**Review by noah10 — APPROVED** [review]

**Pull request merged**